packages/polymath-offchain/src/startup/index.js

```diff
@@ -44,6 +48,8 @@ const connectToNetwork = async (networkId: string, options: NetworkOptions) => {
     if (retries >= maxRetries) {
       if (optional) {
         logger.warn(message);
+        break;
+        q;
```

Comment:
I'm guessing this `q;` was accidental ;p

Reply:
Most definitely